specification/maps/data-plane/Weather/preview/1.0/weather.json

```diff
@@ -2443,6 +2620,262 @@
           "description": "A textual explanation that can be used for display purposes to summarize the index value and category. For example, when the index value for Flight Delays is very good, the description will be \"Conditions are excellent for flying!\"."
         }
       }
+    },
+    "DailyHistoricalActualsResponse": {
+      "type": "object",
+      "properties": {
+        "historicalActuals": {
+          "type": "array",
+          "description": "Historical actuals for each requested day.",
+          "items": {
+            "$ref": "#/definitions/DailyHistoricalActuals"
+          }
+        }
+      }
+    },
+    "DailyHistoricalActuals": {
+      "type": "object",
+      "properties": {
+        "date": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Date and time of the current observation displayed in ISO 8601 format, for example, 2019-10-27T19:39:57-08:00."
+        },
+        "temperature": {
+          "$ref": "#/definitions/WeatherUnitMaxMinAvg",
+          "description": "Temperature values."
+        },
+        "degreeDaySummary": {
+          "$ref": "#/definitions/DegreeDaySummary",
+          "description": "Summary of heating or cooling degree day information"
```

Comment:
Please change the description for degreeDaySummary to:  

Summary of heating or cooling degree day information. Degree days are measures of how cold or warm a location is. A degree day compares the mean (the average of the high and low) outdoor temperatures recorded for a location to a standard temperature of 65 degrees F/ 18 degree C.